index.js

```diff
@@ -0,0 +1,79 @@
+// create a `validatePassword` function which will return true when 
+// the password given meets all the listed requirements listed below.
+
+const validatePassword = (valPass) => {
+    
```

Comment:
You already declared validatePassword down below so this one made that one null and void and broke your code